src/main/scala/org/allenai/plugins/CoreRepositories.scala

```diff
@@ -28,8 +24,7 @@ object CoreRepositories {
     val defaults = Seq(
       spray,
       Resolver.jcenterRepo,
-      typesafeReleases,
-      ai2PublicReleases
```

Comment:
That one is still there.  It shows up as a remove here because the trailing comma was removed on this line.